package Echo;

option cc_generic_services = true;

message EchoRequest
{
    required int32 code = 1;
};

message EchoResponse
{
    required int32 code = 1;
};

service EchoService
{
    rpc Echo(EchoRequest) returns(EchoResponse);
};
